### DISCUSSION TITLE
Feedback about the new search

### DISCUSSION THREAD

**Discussion by andreynering** [community/community#42129] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

* I miss a sorting mechanism
  * By most recently indexed first (was available in the legacy)
    * Could maybe alternatively be "file added" and/or "file edited" instead
  * More stars first
  * Etc
* I miss some information in the results
  * The last index date was shown in the old version but was removed even in the current "legacy" view. I miss it
    * Could maybe alternatively be "file added" and/or "file edited" instead
  * Showing the number of stars of each repo would be useful as well. [Sourcegraph](https://sourcegraph.com/) does.

Use case explanation: in short, I use the search to to discover which projects are using some of the projects I maintain.

**Comment by look** [comment, marked as answer]
Thanks for your feedback. We've heard from several other people that they miss the "recently indexed" sort option. We removed it because we rebuild the index very frequently so it wouldn't be useful. I think what would be really useful is to sort based on the commit date the file appeared in, but we don't have plans to implement that yet.

**Reply by andreynering** [reply]
Right, thanks for the response.

I'll be looking for the sort support in the future.

In the meantime, you could consider re-adding to the UI the info you removed, as I said. Some file timestamp, specially. Number of stars would be useful as well. They were shown before.

**Comment by oasissoman** [comment]
> Thanks for your feedback. We've heard from several other people that they miss the "recently indexed" sort option. We removed it because we rebuild the index very frequently so it wouldn't be useful. I think what would be really useful is to sort based on the commit date the file appeared in, but we don't have plans to implement that yet.

**Reply by grandmas-favorite** [reply]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like duplicating existing answers. As a result, we'll be hiding this comment. Please note any future violations may result in an indefinite block from the Community. Thanks for understanding.

**Comment by ShHaWkK** [comment]
Thanks for your feedback. We've heard from several other people that they miss the "recently indexed" sort option. We removed it because we rebuild the index very frequently so it wouldn't be useful. I think what would be really useful is to sort based on the commit date the file appeared in, but we don't have plans to implement that yet.

**Reply by grandmas-favorite** [reply]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like duplicating existing answers. As a result, we'll be hiding this comment. Please note any future violations may result in an indefinite block from the Community. Thanks for understanding.
